src/hexen/r_things.c

```diff
@@ -660,6 +667,14 @@ void R_ProjectSprite(mobj_t * thing)
     }
 
     vis->brightmap = R_BrightmapForSprite(thing->state - states);
+#ifdef CRISPY_TRUECOLOR
+    if (thing->flags & (MF_SHADOW | MF_ALTSHADOW))
+    {
+        // [crispy] not using additive blending (I_BlendAdd) here 
+        // to preserve look & feel of original Hexen's translucency
+        vis->blendfunc = I_BlendOverTinttab;
```

Comment:
Might be slightly incorrect, but I was never able to see the difference between `MF_SHADOW` and `MF_ALTSHADOW`. Is second one more opaque or something?

Reply:
ALTSHADOW is slightly more opaque. Here's chocolate with an ettin set to SHADOW:
![HEXEN00](https://github.com/fabiangreffrath/crispy-doom/assets/47063032/97753fdc-fcc3-48d7-99a1-4429eb6310fb)

And now ALTSHADOW:
![HEXEN01](https://github.com/fabiangreffrath/crispy-doom/assets/47063032/218e012f-6c31-4855-b47a-057ac4ed8818)